Calendr/Assets/de.lproj/Localizable.strings

```diff
@@ -86,10 +86,10 @@
 "reminder.options.complete" = "Erledigt";
 "reminder.options.remind" = "Erinnerung %@";
 
-"event_details.participant.organizer" = "Veranstalter";
-"event_details.participant.me" = "mich";
+"event_details.participant.organizer" = "Veranstaltende Person";
+"event_details.participant.me" = "Ich";
 
-"event_status.label" = "mein status:";
+"event_status.label" = "Mein status:";
```

Comment:
This one was lowercase on purpose, to align with the native Calendar UI.

Reply:
Apple Calendar is starting with capital letter as well:

<img width="284" alt="Bildschirmfoto 2024-11-11 um 16 47 03" src="https://github.com/user-attachments/assets/9c31cb91-7d8e-4992-8f43-a25121de3172">

Reply:
geez, german is confusing 🤣